go/teams/handler.go

```diff
@@ -385,10 +385,11 @@ func handleSBSSingle(ctx context.Context, g *libkb.GlobalContext, teamID keybase
 		if err != nil {
 			return err
 		}
+		var ityp string
 		switch category {
 		case keybase1.TeamInviteCategory_SBS:
 			//  resolve assertion in link (with uid in invite msg)
-			ityp, err := invite.Type.String()
+			ityp, err = invite.Type.String()
```

Comment:
`ityp` will not be populated for EMAIL/PHONE, maybe we can move this code to outside of `switch` (before) ?

Reply:
(it's used in `SendChatSBSResolutionMessage` call)